src/builtins/math.cpp

```diff
@@ -1,302 +0,0 @@
-// Implementation of the math builtin.
-#include "config.h"  // IWYU pragma: keep
```

Comment:
please put the deletion in the commit that obsoletes the file.
That makes recursive blame easier.

Might need to combine math + tinyexpr changes to one commit but that's fine.